torch/csrc/export/pybind.cpp

```diff
@@ -15,13 +17,34 @@ void initExportBindings(PyObject* module) {
 
   exportModule.def(
       "deserialize_exported_program", [](const std::string& serialized) {
-        return nlohmann::json::parse(serialized).get<ExportedProgram>();
+        auto parsed = nlohmann::json::parse(serialized);
+
+        // Query the current Python schema version as target
+        py::module_ schema_module =
+            py::module_::import("torch._export.serde.schema");
```

Comment:
I wonder if we could add the schema version to [`generated_serialization_types.h`](https://fburl.com/code/yijogwgt) so that we can query the schema version directly in the `upgrade` cpp function rather than here. cc @zhxchen17 

can leave as a TODO for now

Reply:
> edit: oh, you mean we expose the schema_version in generated_serialization_types.h as a constant? 

yup